src/main/java/com/ecom/shopping/adminkart/AdminKartApplication.java

```diff
@@ -1,12 +1,19 @@
 package com.ecom.shopping.adminkart;
 
+import com.ecom.shopping.adminkart.dto.ProductDto;
+import org.apache.ibatis.type.MappedTypes;
+import org.mybatis.spring.annotation.MapperScan;
+import com.ecom.shopping.adminkart.controller.ProductCategoryController;
+
+import org.apache.ibatis.type.MappedTypes;
+import org.mybatis.spring.annotation.MapperScan;
 import com.ecom.shopping.adminkart.controller.ProductCategoryController;
 import org.apache.ibatis.type.MappedTypes;
 import org.mybatis.spring.annotation.MapperScan;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@MappedTypes({ProductCategoryController.class})
+@MappedTypes(value = {ProductCategoryController.class, ProductDto.class})
```

Comment:
try to use only package name apart from DTo class and if working fine then use package name

Reply:
without Dto class not working ,please go through below link to check issues.
```
![Screenshot (332)](https://user-images.githubusercontent.com/77046813/226281510-1918b84c-c58a-495c-9f19-39105a6421af.png)